manual_build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+GORELEASER_IMAGE=iotmod/goreleaser-cgo-cross-compiler:1.13.5
+REPO_PATH="/build"
+PWD="$(pwd)"
+
+mkdir -p ${PWD}
```

Comment:
Always good to use double quotes on variables in bash unless explicitly not required (e.g. multiple cli options provided variable)